src/settings.cc

```diff
@@ -244,11 +242,10 @@ Napi::Value getTraceSettings(const Napi::CallbackInfo& info) {
     return o;
   }
 
+  have_metadata = in.in_xtrace != nullptr;
```

Comment:
Please remove line 200 `in.in_xtrace = xtrace.c_str();`. The value will be overwritten by line 207 logic.

Reply:
Indeed. 

Done in [64b8291](https://github.com/appoptics/appoptics-bindings-node/pull/118/commits/64b82915ae6057e80a5b79f06d7f2593104e8ff0). Tested.